tox.ini

```diff
@@ -19,7 +19,14 @@ description = run the tests with pytest under {basepython}
 setenv = PIP_DISABLE_VERSION_CHECK = 1
          COVERAGE_FILE = {env:COVERAGE_FILE:{toxworkdir}/.coverage.{envname}}
          VIRTUALENV_NO_DOWNLOAD = 1
-passenv = http_proxy https_proxy no_proxy SSL_CERT_FILE PYTEST_*
+passenv =
+    CURL_CA_BUNDLE
```

Comment:
why would you need it to set it there if it's done internally 🤔

Reply:
Because the tox version used to test tox itself is an older version, one that is not able to
pass these vars. I had to do this locally in order to be able to run tox-on-tox. If I would
install from source before it could have worked.